### ISSUE TITLE
(v2) token info query returns null for AutoRenewAccount

### ISSUE THREAD

**Issue by gregscullard** [hashgraph/hedera-sdk-js#293]
```
resp = await new TokenCreateTransaction()
        .setNodeAccountIds([resp.nodeId])
        .setTokenName("ffff")
        .setTokenSymbol("F")
        .setDecimals(2)
        .setInitialSupply(0)
        .setTreasuryAccountId(client.operatorAccountId)
        .setAutoRenewAccount(client.operatorAccountId)

...

const tokenInfo = await new TokenInfoQuery()
        .setTokenId(tokenId)
        .execute(client);

console.log(`token info ${tokenInfo.autoRenewAccount}`);
```

==> token info null

**Comment by mehcode** [comment]
`setAutoRenewAccount` should be `setAutoRenewAccountId` and `tokenInfo.autoRenewAccount` should be `tokenInfo.autoRenewAccountId`

**Comment by danielakhterov** [comment]
Renamed and fixed in v2.0.2 release
